src/python_info.rs

```diff
@@ -49,5 +56,9 @@ fn get_python_binary(requested_python: &Option<String>) -> Result<std::path::Pat
     }
 
     // Python3 may be called 'python', for instance on Windows
-    Ok(which::which("python")?)
+    let res = which::which("python");
+    if res.is_err() {
+        return error::new("Neither `python3` nor `python` fonud in PATH");
+    }
+    Ok(res.unwrap())
```

Comment:
Same idea with `map_err`, that's maybe just me but I find it a little more idiomatic!

```rust
which::which("python").map_err(|e| Error::Other { 
    message: "Neither `python3` nor `python` fonud in PATH".to_string() 
})
```

Reply:
Agree. Let's try that